scss/utilities/rounded-corners-mixin.scss

```diff
@@ -11,13 +11,29 @@
   border-image-source: url('data:image/svg+xml;utf8,<?xml version="1.0" encoding="UTF-8" ?><svg version="1.1" width="5" height="5" xmlns="http://www.w3.org/2000/svg"><path d="M2 1 h1 v1 h-1 z M1 2 h1 v1 h-1 z M3 2 h1 v1 h-1 z M2 3 h1 v1 h-1 z" fill="rgb(#{red($color)},#{green($color)},#{blue($color)})" /></svg>');
 }
 
+@mixin border-image-repeat() {
+  border-image-repeat: stretch;
+
+  // for chrome
+  @media all and (-webkit-min-device-pixel-ratio: 0) and (min-resolution: 0.001dpcm) {
+    border-image-repeat: space;
+  }
+
+  // for firefox
+  @supports (-moz-appearance: meterbar) {
+    border-image-repeat: stretch;
+  }
+}
+
 @mixin rounded-corners($isDark: false) {
   @extend %rounded-corner-defaults;
 
   border-image-slice: 3;
   border-image-width: 3;
   border-image-repeat: stretch;
```

Comment:
It is unnecessary because it moved to `mixin border-image-repeat`. 👍 
```suggestion
```

Reply:
I'm sorry my mistake